extensions/ql-vscode/src/contextual/fileRangeFromURI.ts

```diff
@@ -20,9 +20,8 @@ export default function fileRangeFromURI(uri: UrlValue | undefined, db: Database
       Math.max(0, (loc.endLine || 0) - 1),
       Math.max(0, (loc.endColumn || 0)));
     try {
-      const parsed = vscode.Uri.parse(uri.uri, true);
-      if (parsed.scheme === 'file') {
-        return new vscode.Location(db.resolveSourceFile(parsed.fsPath), range);
+      if (uri.uri.startsWith('file')) {
```

Comment:
Can we make this `file:` to prevent silly uris like `filefoo:` from crashing.

Reply:
sure.